### DISCUSSION TITLE
In the case of display='spinner , we wouldn't like to use 'textinput'.

### DISCUSSION THREAD

**Discussion by KimJeonghun91** [mmazzarolo/react-native-modal-datetime-picker#686] [Q&A]
In the case of display='spinner', if inner data of the DataTimePickerModal is 'onPress', 'softkeyboard' appears on the screen.

We would like to disable the function. But, we would like only to keep using a scroll function.
Would you please let me know how to solve it?

**Comment by mmazzarolo** [comment]
Does it also happen with [@react-native-community/datetimepicker](https://github.com/react-native-community/react-native-datetimepicker)?
